src/containers/EnterpriseApp/__snapshots__/EnterpriseApp.test.jsx.snap

```diff
@@ -41,159 +41,4 @@ exports[`EnterpriseApp renders error page correctly 1`] = `
-              viewBox="0 0 640 512"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path
-                d="M497.941 225.941L286.059 14.059A48 48 0 0 0 252.118 0H48C21.49 0 0 21.49 0 48v204.118a48 48 0 0 0 14.059 33.941l211.882 211.882c18.744 18.745 49.136 18.746 67.882 0l204.118-204.118c18.745-18.745 18.745-49.137 0-67.882zM112 160c-26.51 0-48-21.49-48-48s21.49-48 48-48 48 21.49 48 48-21.49 48-48 48zm513.941 133.823L421.823 497.941c-18.745 18.745-49.137 18.745-67.882 0l-.36-.36L527.64 323.522c16.999-16.999 26.36-39.6 26.36-63.64s-9.362-46.641-26.36-63.64L331.397 0h48.721a48 48 0 0 1 33.941 14.059l211.882 211.882c18.745 18.745 18.745 49.137 0 67.882z"
-                fill="currentColor"
-                style={Object {}}
-              />
-            </svg>
-            <span
-              className="sr-only"
-            >
-              Code Management
-            </span>
-          </a>
-        </li>
-        <li
-          className="nav-item"
-        >
-          <a
-            aria-current={null}
-            className="nav-link text-left rounded-0"
-            href="//admin/subscriptions"
-            onClick={[Function]}
-          >
-            <svg
-              aria-hidden="true"
-              className="svg-inline--fa fa-credit-card fa-w-18 "
-              data-icon="credit-card"
-              data-prefix="fas"
-              focusable="false"
-              role="img"
-              style={Object {}}
-              viewBox="0 0 576 512"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path
-                d="M0 432c0 26.5 21.5 48 48 48h480c26.5 0 48-21.5 48-48V256H0v176zm192-68c0-6.6 5.4-12 12-12h136c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12H204c-6.6 0-12-5.4-12-12v-40zm-128 0c0-6.6 5.4-12 12-12h72c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12H76c-6.6 0-12-5.4-12-12v-40zM576 80v48H0V80c0-26.5 21.5-48 48-48h480c26.5 0 48 21.5 48 48z"
-                fill="currentColor"
-                style={Object {}}
-              />
-            </svg>
-            <span
-              className="sr-only"
-            >
-              Subscription Management
-            </span>
-          </a>
-        </li>
-      </ul>
-    </div>
-  </nav>
-  <div
-    className="content-wrapper"
-    style={
-      Object {
-        "paddingLeft": 10,
-      }
-    }
-    tabIndex="-1"
-  >
-    <main
-      role="main"
-    >
-      <div
-        className="container-fluid mt-3"
-      >
-        <div
-          className="text-center py-5"
-        >
-          <h1>
-            404
-          </h1>
-          <p
-            className="lead"
-          >
-            Oops, sorry we can't find that page!
-          </p>
-          <p>
-            Either something went wrong or the page doesn't exist anymore.
-          </p>
-        </div>
-      </div>
-    </main>
-  </div>
-</div>
-`;
+exports[`<EnterpriseApp /> renders not found page correctly 1`] = `null`;
```

Comment:
Is it expected that this snapshot is now resulting in `null` vs. the 404? This snapshot is for the case when the `enterpriseSlug` is missing from the route I believe.

If so, should we remove that test altogether?

Reply:
I'm in favor of removing snapshot tests in general. I'll look into adding a unit test that tests the right thing.

Reply:
I do not like snapshot tests! I think once we have some cypress tests, and well written hooks and ui tests, we should feel less of a need for these (or no need hopefully)